<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ADMIN-PC" ignored="0" name="Test" tests="5" failures="1" timestamp="15 Aug 2020 05:05:09 GMT" time="16.908" errors="0">
  <testcase name="verifyTitle" time="0.012" classname="com.demo.negativeTest"/>
  <testcase name="step1" time="0.163" classname="com.demo.negativeTest"/>
  <testcase name="step2" time="0.369" classname="com.demo.negativeTest"/>
  <testcase name="step3" time="4.944" classname="com.demo.negativeTest">
    <failure type="java.lang.AssertionError" message="The interest rate is not 0 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The interest rate is not 0 expected [true] but found [false]
at com.demo.negativeTest.step3(negativeTest.java:119)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- step3 -->
  <testcase name="step4" time="0.118" classname="com.demo.negativeTest"/>
</testsuite> <!-- Test -->
